catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -269,10 +273,22 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       throw new UnsupportedQueryException("Could not complete solr query.", e);
     }
 
-    responseProps.put("query", query.get("q"));
     return new SourceResponseImpl(request, responseProps, results, totalHits);
   }
 
+  private List<String> getSearchTermFieldValues(QueryResponse solrResponse) {
+    Map<String, Suggestion> suggestions = solrResponse.getSpellCheckResponse().getSuggestionMap();
+    Set<String> fieldValues = suggestions.keySet();
+    removeResourceFieldValue(fieldValues);
```

Comment:
✏️ I don't think we need to perform the check. If the set does not contain the string it just won't remove it.

Reply:
This is also a good candidate for a comment explaining why it needs to be done.

Reply:
The resource fields get automatically appended in the query. It's not something the user actually put in